practice_app/pages/api/getRedditData.test.js

```diff
@@ -0,0 +1,17 @@
+const { checkGetRedditData,getPostData  } = require('./getRedditDataModule')
+const testResponse = require('./exampleRedditData.json')
+
+describe('checkGetRedditData', () => {
```

Comment:
I think existing tests are good tests but I suggest you also add some tests for a malformed input data. If I understand it correctly, you check `getPostData` function for a well formed json input in `./exampleRedditData.json`. Maybe you can add 2-3 more cases where different parts of the input is malformed or missing.